java/src/generated/java/io/cucumber/messages/types/TestCase.java

```diff
@@ -36,6 +39,10 @@ public java.util.List<TestStep> getTestSteps() {
         return testSteps;
     }
 
+    public Optional<String> getTestRunStartedId() {
+        return Optional.ofNullable(testRunStartedId);
```

Comment:
What  is the rationale for making the field optional? I don't think there is a sensible behavior for a consumer that needs this field and doesn't get it.

Reply:
If we make it required now, the new code will be immediately incompatible with messages generated from the old code. This means e.g. we'd need to wait for all implementations to add this before we could use the new messages version in formatters.

If we're fine with that then I'm happy to change. But it seemed better to add it as optional, and maybe circle back and make it required down the line once it's established everywhere.

Reply:
Okay, that's acceptable. Can you make a ticket so we won't forget to make it required?

Reply:
Done https://github.com/cucumber/messages/issues/113